src/abstractdataframe/io.jl

```diff
@@ -47,15 +47,8 @@ Base.show(io::IO, mime::MIME"text/csv", df::AbstractDataFrame) =
     printtable(io, df, header = true, separator = ',')
 Base.show(io::IO, mime::MIME"text/tab-separated-values", df::AbstractDataFrame) =
     printtable(io, df, header = true, separator = '\t')
-Base.show(io::IO, mime::MIME"text/plain", df::AbstractDataFrame;
-          allrows::Bool = !get(io, :limit, false),
-          allcols::Bool = !get(io, :limit, false),
-          splitcols = get(io, :limit, false),
-          rowlabel::Symbol = :Row,
-          summary::Bool = true,
-          eltypes::Bool = true) =
-    show(io, df, allrows=allrows, allcols=allcols,
-         splitcols=splitcols, rowlabel=rowlabel, summary=summary, eltypes=eltypes)
+Base.show(io::IO, mime::MIME"text/plain", df::AbstractDataFrame; kwargs...) =
```

Comment:
why this `show` accepts any `kwargs` and the `show` below only a limited list of keyword arguments?

Reply:
The `kwargs` are the arguments that can be passed to `pretty_table`. We are using PrettyTables.jl only for text printing, this is why only this `show` has `kwargs`. Did I understand the question correctly?

Reply:
I meant `show` for other types, like `DataFrameRows` below (in the other file). You did not allow passing `kwargs` there.

Reply:
In general if I remember correctly we provide `show` for:
1. `AbstractDataFrame`
2. `DataFrameRow`
3. `DataFrameRows`
4. `DataFrmeCols`
5. `GroupedDataFrame`

and it would be good to have a consistent API for them

Reply:
Hum! Good, this seems a bug in my code, I will fix it.